django_opentracing/tracer.py

```diff
@@ -1,12 +1,26 @@
 from django.conf import settings
 import opentracing
 
+django_tracer = None
+
+def get_tracer():
+    return opentracing.tracer
+
+def get_current_span(request):
+    # this lets django rest framework work seamlessly since they wrap the request
+    if hasattr(request, '_request'):
+        request = _request
+    return django_tracer.get_span(request)
+
+
```

Comment:
So far we had been saving the span for each request in a dictionary, as part of the `DjangoTracer`. Is there any *need* to change this behaviour?